nextest-runner/src/config.rs

```diff
@@ -283,11 +284,58 @@ struct DefaultProfileImpl {
     failure_output: TestOutputDisplay,
     success_output: TestOutputDisplay,
     fail_fast: bool,
-    #[serde(with = "humantime_serde")]
-    slow_timeout: Duration,
+    #[serde(deserialize_with = "deserialize_slow_timeout")]
+    slow_timeout: SlowTimeout,
     junit: DefaultJunitImpl,
 }
 
+/// Type for the slow-timeout config key.
+#[derive(Clone, Debug, Deserialize, Default)]
```

Comment:
Can this be `Copy`?

Reply:
instead of `Clone` or in addition?

Reply:
It would be in addition -- `Copy` requires (derived) `Clone`.